healthcare-concept-chain/src/healthcare-chain.ts

```diff
@@ -190,7 +260,7 @@ export class HealthcareConceptRefineChain extends BaseChain {
       llm: fields.llm,
     });
 
-    this.txEndpoint = fields.txEndpoint ?? this.txEndpoint;
+    this.txEndpoint = fields.txEndpoint ?? endpoints[1];
```

Comment:
```suggestion
    this.txEndpoint = fields.txEndpoint ?? endpoints["default"];
```

I think this should be default if I am following correctly?